packages/ckeditor5-typing/tests/deleteobserver.js

```diff
@@ -261,6 +261,107 @@ describe( 'DeleteObserver', () => {
 	}
 } );
 
+describe( 'DeleteObserver - handling Shift + Delete on Windows', () => {
```

Comment:
Maybe it would be good to test backspace whether it's not affected. Also, the tests for non Windows would be good to confirm that this modified behavior is not affecting other platforms.

Reply:
Added one new unit test for non-Windows platform and 2 tests for <kbd>Backspace</kbd> handling for `DeleteObserver`.